### ISSUE TITLE
"connection refused" when connecting to iOS device

### ISSUE THREAD

**Issue by stranbird** [frida/frida-core#12]
I was once successfully got it right for `frida-ps -U`, when I tried to open an app and hooked up again, the "connection refused" error came up. Subsequent commands failed as well.
```
> $ frida-ps -U
Failed to enumerate processes: connect failed (connection refused)
```

My settings:

iPad Mini with iOS 8.1.2
OSX 10.10
Frida 1.6.8

Thanks!

**Comment by stranbird** [comment]
Moving from frida/frida-website#13

**Comment by oleavr** [comment]
This should now be fixed in latest git thanks to @Tyilo who tracked this down and fixed it in 7952b52a942e2fcff7e23f081a27fabfbfa0e912.
